ci-automation/release.sh

```diff
@@ -93,7 +93,11 @@ function _inside_mantle() {
     secret_to_file google_release_credentials_file "${GOOGLE_RELEASE_CREDENTIALS}"
 
     for platform in aws azure; do
-      for arch in amd64 arm64; do
+      arches=(amd64 arm64)
+      if [[ "${CHANNEL}" == "lts" ]] && [[ "${platform}" == "azure" ]] && echo "$VERSION" | grep -q "^3033"; then
```

Comment:
Forgot about one thing:

```suggestion
      # TODO: drop when LTS-3033 is not supported any more.
      if [[ "${CHANNEL}" == "lts" ]] && [[ "${platform}" == "azure" ]] && echo "$VERSION" | grep -q "^3033"; then
```

Reply:
No, nevermind, I thought this is a part of the `main` branch, but it's actually for `flatcar-3033`.

Reply:
Ah, didn't see that either - I would suggest to have it on the `main` branch to have more clarity on what code runs where